src/main/java/de/zalando/aruha/nakadi/service/subscription/StreamingContext.java

```diff
@@ -160,7 +160,7 @@ private void rebalance() {
             clientListChanges.refresh();
             zkClient.runLocked(() -> {
                 final Partition[] changeset = rebalancer.apply(zkClient.listSessions(), zkClient.listPartitions());
-                if (changeset != null && changeset.length > 0) {
```

Comment:
I changed apply method so that it will never return null
https://github.com/zalando/nakadi/pull/330/files#diff-9797c80ff4effa509ddb92a7eda3c526R68
So there is no need to check additionally for null